<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticalMonitoringVersionWidget</class>
 <widget class="QWidget" name="StatisticalMonitoringVersionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="6">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1,0,1,0">
     <item>
      <widget class="QComboBox" name="comboBoxColumToFilter"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditFrom">
       <property name="toolTip">
        <string>ISODate</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>to: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditTo">
       <property name="toolTip">
        <string>ISODate</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonRefresh">
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../gex.qrc">
       <normaloff>:/gex/icons/refresh.png</normaloff>:/gex/icons/refresh.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>550</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonImport">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="icon">
      <iconset resource="../../gex.qrc">
       <normaloff>:/gex/icons/Import.png</normaloff>:/gex/icons/Import.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Download this SPM version</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="../../gex.qrc">
       <normaloff>:/gex/icons/trash.png</normaloff>:/gex/icons/trash.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../gex.qrc"/>
 </resources>
 <connections/>
</ui>
